server/src/main/java/org/opensearch/gateway/remote/RemotePersistenceStats.java

```diff
@@ -46,4 +49,12 @@ public void indexRoutingFilesCleanupAttemptFailed() {
     public long getIndexRoutingFilesCleanupAttemptFailedCount() {
         return indexRoutingFilesCleanupAttemptFailedCount.get();
     }
+
+    public void indicesRoutingDiffFileCleanupAttemptFailed() {
+        indexRoutingFilesCleanupAttemptFailedCount.incrementAndGet();
+    }
+
+    public long getIndicesRoutingDiffFileCleanupAttemptFailedCount() {
+        return indexRoutingFilesCleanupAttemptFailedCount.get();
+    }
```

Comment:
Did we mean to add `indicesRoutingDiffFilesCleanupAttemptFailedCount` here?

Reply:
Yes, addressed in separate PR.